shared/crypto/output/index.tsx

```diff
@@ -250,12 +260,59 @@ export const OutputBar = (props: OutputBarProps) => {
   )
 }
 
+const OutputFileDestination = (props: {operation: Types.Operations}) => {
+  const {operation} = props
+  const operationTitle = capitalize(operation)
+  const dispatch = Container.useDispatch()
+
+  // State
```

Comment:
i really don't think these types of comments really add value, just noise

Reply:
It's nice to organize things for when others come to refactor. I can remove.